package.json

```diff
@@ -31,23 +33,19 @@
   "devDependencies": {
     "@commitlint/cli": "^16.2.3",
     "@commitlint/config-conventional": "^16.0.0",
+    "@jsdevtools/host-environment": "^2.1.2",
+    "@jsdevtools/karma-config": "^3.1.7",
     "@readme/eslint-config": "^8.5.1",
     "@readme/oas-examples": "^5.0.0",
-    "alex": "^10.0.0",
-    "datauri": "^4.1.0",
```

Comment:
Dumped the `datauri` package because it requires `fs` which we obviously can't use in the browser. We were only using it for encoding `owlbert.png` into a data URL for file upload tests so I've instead encoded that myself and saved it to a `owlbert.dataurl.json` fixture. I had to do this same thing when I set up browser testing in [fetch-har](https://npm.im/fetch-har).